src/adapters/auth.ts

```diff
@@ -0,0 +1,25 @@
+import axios, { AxiosResponse } from 'axios'
```

Comment:
@gutakk We already have a file to handle the Axios requests - `lib/requestManager.ts`. Can't we use that?

Reply:
Thanks for pointing this out.
Fixed in 46305f348020b077c5769afd7ff4e76f7573c021

Actually, I didn't know we have this in the template.